packages/system-tests/scenarios/forceSoqlQuery.test.ts

```diff
@@ -86,6 +86,7 @@ describe('force:soql:query commands', () => {
     await app.wait();
 
     // TODO Select all text in current active window
+    await app.client.keys(['Cmd', 'a'], false);
```

Comment:
I tried different options here like Meta, Cmd etc, but without success. Do you have an idea how to add support for Ctrl+A?

Reply:
Not sure which OS you are working on but I was able to get it running with `await app.client.keys(['Meta', 'a', 'NULL'], false);` on my Mac. However, if you look at the `command()` and `transliterate()` methods in `application.ts`, you should be able to do Ctrl+A with `await app.command('ctrl+a')`.

Reply:
Once you get it working locally, please make this test agnostic to Windows/Linux/Mac by checking the os platform so this will pass on all the builds